java/src/jni/h5util.c

```diff
@@ -4043,6 +4043,551 @@ Java_hdf_hdf5lib_H5_H5export_1attribute(JNIEnv *env, jclass clss, jstring file_e
         H5Aclose(attr_id);
 } /* end Java_hdf_hdf5lib_H5_H5export_1attribute */
 
+herr_t
+translate_rbuf(JNIEnv *env, jobjectArray ret_buf, jlong mem_type_id, H5T_class_t type_class, jsize count,
+               jobjectArray raw_buf)
```

Comment:
Same here that raw_buf should probably be marked as a `void *` or similar since it's just a byte buffer and not a jobjectArray.

Reply:
Sounds right